### PR TITLE
the 'recursive' definition

### PR DESCRIPTION
**Pull request by gocko**
A combination of a message and its receiver is called a message is not consistent with the figure. There it is called 'a message send' if I am right.

### CODE REVIEW

**Comment by gocko** [comment]
I can check other similar inconsistencies if this is ok. I think I've seen some more...

File: Chapters/UnderstandingMessage/UnderstandingMessage.pillar [hunk 1]
```diff
@@ -10,7 +10,7 @@ The Pharo's syntax is closed to the one of Smalltalk, so Smalltalk programmers c
 !!!Identifying messages
 
 In Pharo, except for the syntactic elements listed in Chapter *@chapterSyntaxNutshell* (\:= ^ . ; # () {} [ : | ]), everything is a message send.
-As in C\+\+, you can define operators like ==\+== for your own classes, but all operators have the same precedence.
+You can define operators like ==\+== for your own classes, but all operators, exsiting and defined ones, have the same precedence.
```
Comment (cdlm) [inline Chapters/UnderstandingMessage/UnderstandingMessage.pillar]: typo! existing

**Comment by Ducasse** [comment]
Color yellow -> Creates a color yellow
Composite -> composed because we never mentioned elsewhere composite and I would prefer to keep composed.

**Comment by gocko** [comment]
ok. done!

**Comment by gocko** [comment]
Another thing that might be fixed is that in examples and in the table we talk about kinds of messages before they are introduced and explained. Is that intentionally?

**Comment by Ducasse** [comment]
Probably a mistake :)

**Pull request merged**